packages/@aws-cdk/aws-s3/test/test.bucket.ts

```diff
@@ -72,6 +72,106 @@ export = {
     test.done();
   },
 
+  'valid bucket names'(test: Test) {
+    const stack = new cdk.Stack();
+
+    test.ok(() => new s3.Bucket(stack, 'MyBucket1', {
```

Comment:
Turns out `ok` does not fail if there is, in fact, an exception thrown. I'll be switching this around to use `doesNotThrow`